### PR TITLE
Finished documentation, added additional HRA and HRV parameters

### PR DESCRIPTION
**Pull request by maria-guzik**
Added and documented paramerters:
-HRA1, HRA2, HRAN, HRAT, HRAcomp
-Nd (Porta's index), SD2/SD1, pNN50
-CS, CSa, CSd, CLa, CLd

### CODE REVIEW

File: signal_properties/Poincare.py [hunk 1]
```diff
@@ -320,21 +320,21 @@ def total_asymmetry(self):
...
         failed = False
         try:
             hra1 = 1 if self.C1d > 0.5 else 0
             hra2 = 1 if self.C2d < 0.5 else 0
             hrat = 1 if self.Cd < 0.5 else 0
-            hran = 1 if self.C1d > 0.5 else 0 #FIX
+            hran = 1 if self.ND < 0.5 else 0
             hracomp = 1 if (hra1 + hra2) == 2 else 0
         except TypeError:
             failed = True
```
Comment (jaropis) [inline signal_properties/Poincare.py:347]: Great idea to add these! A short point: please make separate PR's for extra tasks. So one PR should be for the documentation, and the other for the extra parameters. Ohter than that, great work. And really good  programing style in this method!

File: plot_tests.py [hunk 2]
```diff
@@ -51,4 +51,6 @@
 #print(rr.annotation)
 
 
-print(rr.poincare.SD2I, rr.poincare.SD2, rr.poincare.SD1I, rr.poincare.SD1)
+#print(rr.poincare.HRA1, rr.poincare.HRA2, rr.poincare.HRAT, rr.poincare.HRAN, rr.poincare.HRAcomp)
+#print(rr.poincare.SDNN, rr.poincare.SD1, rr.poincare.SD1, rr.poincare.SD2I, rr.poincare.meanRR, rr.poincare.SDNNd, rr.poincare.SDNNa, rr.poincare.SD1d, rr.poincare.SD1a, rr.poincare.SD2a, rr.poincare.ND)
+print(rr.annotation, rr.annotation_filter)
```
Comment (jaropis) [inline plot_tests.py]: We will have to do something about this file, but let's leave it for later.

File: signal_properties/Poincare.py [hunk 3]
```diff
@@ -187,6 +213,17 @@ def cv(self):
...
+    def pnnx(self):
+        '''
+        Calculates the pNN based on the differences between consequtive RR intervals (xi and xii)
+
+        Returns:
+            pNN50 (float): Proportion of consecutive RR intervals of normal (sinus) orign that differ by more than 50ms
+        '''
+        differences = abs(self.xii - self.xi)
+        pnn50 = 100*len(where(differences > 50)[0])/len(self.xi)
+        return pnn50
```
Comment (jaropis) [inline signal_properties/Poincare.py:234]: As it stands this is pnn50 - maybe the x in pnnx should be parameterized, with the default value of 50?

**Review by jaropis — CHANGES_REQUESTED** [review]
Overall great work. As noted in another comment, separate functionality (additional methods) should be addressed in separate PR's - but this is only a minor point. 

Please parameterize pnnx and also either use the word "method" for functions belonging to a class, as requested in the previous code review, or defend your decision to call them "functions" (I am open to a discussion)

File: signal_properties/Poincare.py (hunk 3 above)
Comment (maria-guzik) [reply signal_properties/Poincare.py:234]: This was my plan initially but I wasn't sure if the rest pnnX would be useful

**Comment by maria-guzik** [comment]
Added the requested change, is there any spot in the Poincare class where I still use 'function' instead of 'method'?

File: signal_properties/Poincare.py (hunk 1 above)
Comment (maria-guzik) [reply signal_properties/Poincare.py:347]: If I add additional functionality can I also document it within the same PR?

Comment (jaropis) [reply signal_properties/Poincare.py:347]: Yes - this is what you should do - if you add functionality, document it in the same PR.

File: signal_properties/Poincare.py (hunk 3 above)
Comment (jaropis) [reply signal_properties/Poincare.py:234]: It definitely will be useful!

**Review by jaropis — APPROVED** [review]
well done! ready to be merged
